source/utilities.tex

```diff
@@ -40,7 +40,7 @@
 \indextext{\idxhdr{utility}}%
 \indexlibrary{\idxhdr{utility}}%
 \indexlibrary{\idxcode{rel_ops}}%
-\synopsis{Header \tcode{<utility>} synopsis}
+\rSec2[utility.syn]{Header \tcode{<utility>} synopsis}
```

Comment:
I had thought about this, too, but until now no "Synopsis" section that has a numbered heading has an introductory paragraph. That's why I haven't made this change yet, I wonder if we need some more rewording.